project/models.py

```diff
@@ -5,3 +5,27 @@ class User(db.Model):
     email = db.Column(db.String(100), unique=True)
     password = db.Column(db.String(100))
     name = db.Column(db.String(1000))
```

Comment:
I think this is ok, I was wondering if this addition would pose a conflict to the initial implementation. But it is good o go.

Reply:
I actually just modified what you had already,  not a new code